python/hail/expr/expression.py

```diff
@@ -529,6 +530,157 @@ def __ne__(self, other):
             ))
         return self._bin_op("!=", other, tbool)
 
+    def _to_table(self):
+        source = self._indices.source
+        axes = self._indices.axes
+        if not self._aggregations.empty():
+            raise NotImplementedError('cannot convert aggregated expression to table')
+        if source is None:
+            # scalar expression
+            df = hail.Table.range(1) # wat
+            df = df.select(**{'<expr>': self})
+            return df
+        elif len(axes) == 0:
+            source = source.select_globals(**{'<expr>': self})
+            df = hail.Table.range(1)
+            df = df.select(**{'<expr>': source.view_join_globals()['<expr>']})
```

Comment:
This can just be `globals_table()`, no?

Reply:
globals_table() doesn't exist :(

Reply:
Oops, I didn't add it to table!  My bad.  Noted: https://github.com/hail-is/hail/issues/2970.